Innovative Flix/force-app/main/default/objects/Season__c/fields/Tv_Serie__c.field-meta.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Tv_Serie__c</fullName>
+    <externalId>false</externalId>
+    <label>Tv Serie</label>
+    <referenceTo>Tv_Serie__c</referenceTo>
```

Comment:
Add also formula field to show title of tv serie

Reply:
Done